required/tools/varioref.dtx

```diff
@@ -2344,7 +2344,7 @@
 %    the argument of |\is@pos@num@|. Note, that the added space in
```

Comment:
Since other two "Note, that ..." are already changed to "Note that ...", maybe also

```suggestion
%    the argument of |\is@pos@num@|. Note that the added space in
```

Though to me, a non-native speaker, these two forms both look OK.